src/network/networkd-network.h

```diff
@@ -311,6 +315,7 @@ struct Network {
         Hashmap *prefixes_by_section;
         Hashmap *route_prefixes_by_section;
         Hashmap *rules_by_section;
+        Hashmap *dhcp_static_leases_by_section;
```

Comment:
use an OrderedHashmap if the order matters. Don#t use Hashmap + linked list, that's needlessly redundant